bin/workflow_tools_install.py

```diff
@@ -0,0 +1,22 @@
+import os
+import yaml
+import sys
+
+HEAD = """
+---
```

Comment:
@bedroesb if you have time I think a better way would be to integrate this deeply into shed-install (ephemeris project). Maybe even without conversions. Also please note that the workflow format will change (optionally) to be a yaml file. I'm in favor of going all with this new format and do not include any support for the old ga files here in the training.